roles/fail2ban_exporter/vars/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+go_arch_map:
+  i386: '386'
+  x86_64: 'amd64'
+  aarch64: 'arm64'
+  armv7l: 'armv7'
+  armv6l: 'armv6'
+
+go_arch: "{{ go_arch_map[ansible_architecture] | default(ansible_architecture) }}"
+_gitlab_projectid: 24199687
```

Comment:
I guess we need to update `.github/scripts/version_updater.sh` to handle checking for new versions from GitLab projects.